ORStools/proc/isochrones_layer_proc.py

```diff
@@ -129,14 +148,20 @@ def processAlgorithm(self, parameters, context, feedback):
             if feedback.isCanceled():
                 break
 
-            requests.append({
+            params = {
                 "locations": locations,
                 "range_type": dimension,
                 "range": ranges_proc,
                 "attributes": ['total_pop'],
                 "id": id_value,
-                "options": options
-            })
+                "options": options,
+            }
+
+            if smoothing or smoothing == 0:
+                params["smoothing"] = smoothing
```

Comment:
We're just checking for `None` here, right? I'd suggest the following for clarity then:
```suggestion
            # only include smoothing if set
            if smoothing is not None:
                params["smoothing"] = smoothing
```